src/perf/dbserver/QuicPerformanceDataServer/Controllers/PerformanceController.cs

```diff
@@ -0,0 +1,315 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using QuicDataServer.Data;
+using QuicDataServer.Models;
+using QuicDataServer.Models.Db;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+#pragma warning disable CA2007 // Consider calling ConfigureAwait on the awaited task
+
+namespace QuicDataServer.Controllers
+{
+
+    [ApiController]
+    [Route("[controller]")]
+    public class PerformanceController : ControllerBase
+    {
+        private readonly PerformanceContext context;
+        private readonly ILogger<PerformanceController> logger;
+        private readonly IConfiguration configuration;
+
+        public PerformanceController(ILogger<PerformanceController> logger, PerformanceContext context,
```

Comment:
Can we try to match the style of the native code as much as possible please?

Reply:
For C#, pretty much the standard is this format, and this is what happens when you auto format in VS and VS Code. I can change it, but unlike native code this is pretty standardized across all C# projects.

Reply:
@microsoft/quic what do other folks think? Match default C# or our code style? I tend to prefer all code in the same repo to follow the same style.